stubs/pycocotools/pycocotools/coco_types.pyi

```diff
@@ -0,0 +1,72 @@
+from _typeshed import Incomplete
```

Comment:
This module doesn't exist at runtime, and that's causing another stubtest failure. You should delete this file.

For these "private-to-the-stub" types like the `_Image` `TypedDict`:
- If a type is only used in one file, you should put the definition in that file (that will also make flake8-pyi happier with you).
- If a type is used in multiple files, you should put it in `__init__.py`, and import it from there. You may have to add a few `# noqa` comments if you do that, as flake8-pyi will believe that the private `TypedDict`s (etc) are unused; it won't be able to tell that they're imported in other files.